### PR TITLE
Internal: Merge nested-elements experiment with container [ED-20667]

### PR DESCRIPTION
**Pull request by marcin-el**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

## 1. Problem & Context

Merges the `nested-elements` experiment into `container` experiment, eliminating the separate experiment flag. Nested Tabs and Accordion now depend on container activation rather than a distinct experiment. Simplifies feature management and reduces user confusion.

## 2. What Changed (Where)

| File | Change |
|------|--------|
| `includes/managers/widgets.php` | Removed NestedElementsModule import; moved Nested Tabs/Accordion to 'container' promoted widgets |
| `modules/nested-elements/module.php` | Added `mutable: false, hidden: true`; removed description; now hidden dependency of container |
| `modules/nested-tabs/module.php`, `modules/nested-accordion/module.php` | Changed `is_active()` to check 'container' instead of 'nested-elements' |
| Widget files (tabs, toggle, accordion, nested-tabs, nested-accordion) | Updated `show_in_panel()` to check 'container' instead of 'nested-elements' |
| `modules/ai/connect/ai.php` | Changed feature detection from `get_active_features()['nested-elements']` to `is_feature_active('container')` |
| `modules/atomic-widgets/opt-in/opt-in.php` | Removed NestedElementsModule from imports and OPT_IN_FEATURES |
| Frontend/test files | Replaced `nested-elements` experiment checks with 'container'; simplified test setup |
| `composer.json` | Changed audit config from `audit.ignore` to `policy.advisories.ignore-id` structure |

## 3. How It Works

Nested elements now activate conditionally based on `container` experiment state. All widget visibility, module activation, and feature detection routes through container flag. The nested-elements experiment persists as hidden/immutable dependency metadata, maintaining backward compatibility while surfacing only container to users.

## 4. Risks

**Low risk.** Experiment merging is additive—nested-elements remains active by default and depends on container, preserving existing behavior. Only risk: tests/edge cases checking nested-elements directly may miss regressions if container activation path differs. Mitigate: verify container experiment thoroughly in QA environments.

_Generated by LinearB AI and added by gitStream._
<sub>AI-generated content may contain inaccuracies. Please verify before using.
💡 **Tip:** You can customize your AI Description using **Guidelines** [Learn how](https://docs.gitstream.cm/automation-actions/#describe-changes)</sub>

### CODE REVIEW

File: core/upgrade/upgrades.php [hunk 1]
```diff
@@ -868,6 +868,17 @@ public static function _v_3_26_0() {
...
+	/**
+	 * Upgrade Elementor 3.35.0 - Merge Nested Elements into Container.
+	 *
+	 * @since 3.35.0
+	 * @static
+	 * @access public
+	 */
+	public static function _v_3_35_0() {
+		delete_option( Experiments_Manager::OPTION_PREFIX . 'nested-elements' );
+	}
```
Comment (davseve) [inline core/upgrade/upgrades.php]: upgrade scripts did a lot of mess in the past.
We need to check this manually, actually upgrade a site and see that it works

File: modules/nested-elements/module.php [hunk 2]
```diff
@@ -15,13 +15,10 @@ public static function get_experimental_data() {
...
-				esc_html__( 'Create a rich user experience by layering widgets together inside "Nested" Tabs, etc. When turned on, we’ll automatically enable new nested features. Your old widgets won’t be affected.', 'elementor' ),
-				esc_html__( 'Learn more', 'elementor' )
-			),
 			'release_status' => Experiments_Manager::RELEASE_STATUS_STABLE,
 			'default' => Experiments_Manager::STATE_ACTIVE,
+			'mutable' => false,
+			'hidden' => true,
 			'dependencies' => [
 				'container',
 			],
```
Comment (davseve) [inline modules/nested-elements/module.php:24]: Shouldn't we completely remove this function?

File: core/upgrade/upgrades.php (hunk 1 above)
Comment (marcin-el) [reply core/upgrade/upgrades.php]: this one is already addressed

File: modules/nested-elements/module.php (hunk 2 above)
Comment (marcin-el) [reply modules/nested-elements/module.php:24]: the ticket mentioned < 3.35 so kept it but now in 4.02 we can delete it as this what the plan was - is that correct?

Comment (davseve) [inline modules/nested-elements/module.php:24]: Open a ticket to remove the experiment in 4.5.0
update minimum pro version core

Comment (davseve) [reply modules/nested-elements/module.php:24]: https://elementor.atlassian.net/browse/ED-24862

**Review by davseve — APPROVED** [review]
